client/app/bundles/shared/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,67 @@
+@import "../base.scss";
+
+body {
+  font-family: $font-family;
+}
+
+.fill select{
+  @include selectDropdown(
+    $background: #6D0839,
+    $width: 143px,
+    $height: 49px,
+    $font-size: $font18
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.fill {
+  background:  #6D0839 url('./down.svg') no-repeat right;
+  width: 143px;
+  border-radius: 5px;
+}
+
+.ghost select{
+  @include selectDropdown(
+    $background: rgb(154, 153, 153),
+    $width: 155px,
+    $height: 68px,
+    $font-size: $font22
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.ghost {
+  background:  #9a9999 url('./down.svg') no-repeat right;
+  width: 155px;
+  border-radius: 5px;
+  padding-right: 10px !important;
```

Comment:
Curious why you used `!important` here -- if it's possible to avoid, let's avoid using it! Also, this may be causing a slight UX issue where we're unable to click the very right side of `<DropdownGhost>` components.